.github/workflows/ci.yaml

```diff
@@ -102,6 +102,32 @@ jobs:
            && !contains(github.event.pull_request.body, '[skip test_windows]')
          )"
 
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@v2
+
+      - name: Test
+        run: sbt ";scala3-bootstrapped/compile"
+        shell: cmd
+
+      - name: Scala.js Test
+        run: sbt ";sjsJUnitTests/test ;sjsCompilerTests/test"
+        shell: cmd
+
+  test_windows_full:
+    runs-on: [self-hosted, Windows]
+    if: "github.event_name == 'schedule'
+         || (
+           github.event_name == 'push'
+           && !startsWith(github.event.ref, 'refs/tags/sbt-dotty-')
+         )
+         || (
+           github.event_name == 'pull_request'
+           && !contains(github.event.pull_request.body, '[skip ci]')
+           && !contains(github.event.pull_request.body, '[skip test_windows]')
+           && github.event.pull_request.merged == true
```

Comment:
We don't use `github.event.pull_request.merged == true` for any other job, what is its purpose here? As far as I can see we could handle like we handle `test_sbt`: never run on a PR unless `[test_windows_full]` is present in the PR message.

Reply:
It's intended to run `test_windows_full` only on merged PRs. For open PRs, it will run `test_windows_fast`.

Reply:
For merged PRs, we get a push event to master so we'll run the test that way.

Reply:
Yes, that is simpler.